enable this code only for IA64

strace.c:2259: error: 'PT_CR_IPSR' undeclared (first use in this function)
strace.c:2259: error: (Each undeclared identifier is reported only once
strace.c:2259: error: for each function it appears in.)

diff -Nur strace-4.5.14.orig/strace.c strace-4.5.14/strace.c
--- strace-4.5.14.orig/strace.c	Thu Jan 12 11:50:49 2006
+++ strace-4.5.14/strace.c	Sat Sep 30 09:20:57 2006
@@ -2251,7 +2251,7 @@
 			if (!cflag
 			    && (qual_flags[WSTOPSIG(status)] & QUAL_SIGNAL)) {
 				unsigned long addr = 0, pc = 0;
-#ifdef PT_GETSIGINFO
+#if defined(PT_GETSIGINFO) && defined(IA64)
 #				define PSR_RI	41
 				struct siginfo si;
 				unsigned long psr;
